### ISSUE TITLE
broken packaging

### ISSUE THREAD

**Issue by cleemesser** [cleemesser/python-edf#1]
something has broken the packaging. 
python setup.py install still works
but pip install edflib
fails version 0.82, 0.83

pip version 20.1.1
python 3.7.7

various problems:
tries to re-cython -- errors there
tries to import numpy in setup.py  --- fails though numpy is installed

**Comment by kevinashaw** [comment]
With the following command on MacOS with Py3.7,

`pip install edflib`

I get the following error:

```
AttributeError: type object 'Callable' has no attribute '_abc_registry'
  ----------------------------------------
ERROR: Command errored out with exit status 1: /Users/user/anaconda3/bin/python3.7 /Users/user/anaconda3/lib/python3.7/site-packages/pip install --ignore-installed --no-user --prefix /private/var/folders/5k/q0z0smjj5fz6lcb_t37h52qr0000gn/T/pip-build-env-sy2dlz9y/overlay --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools wheel cython Check the logs for full command output.

```

**Comment by cleemesser** [comment]
Thanks for the info. I need to update my packaging knowledge I have checked that the package still installs from the repo without difficulty. So you can download the source and build from there

**Comment by scott-huberty** [comment]
If the package issue won't be resolved would you consider removing your package from pypl? It can also cause confusion for others trying to install `EDFlib-python`, which  also has the namespace `EDFlib` in python, and is used for example by MNE-Python the EEG/MEG library. if one tries `pip install EDFlib` (not knowing that they should specify `EDFlib-python`) it will by default try (and fail) to install this package.

**Comment by cleemesser** [comment]
packaging issue resolved

**Comment by cleemesser** [comment]
should be resolved with upcoming release of 0.86
